apolo-cli/src/apolo_cli/apps.py

```diff
@@ -0,0 +1,61 @@
+from typing import Optional
+
+from rich.table import Table, box
+
+from .click_types import CLUSTER, ORG, PROJECT
+from .root import Root
+from .utils import argument, command, group
+
+
+@group()
+def app() -> None:
+    """
+    Operations with applications.
+    """
+
+
+@command()
+@argument("cluster", type=CLUSTER, required=False)
+@argument("org", type=ORG, required=False)
+@argument("project", type=PROJECT, required=False)
+async def list(
+    root: Root,
+    cluster: Optional[str],
+    org: Optional[str],
+    project: Optional[str],
+) -> None:
+    """
+    List all apps.
+    """
+    client = root.client
+
+    table = Table(box=box.SIMPLE_HEAVY)
+    table.add_column("ID")
+    table.add_column("Name")
+    table.add_column("Display Name")
+    table.add_column("Template")
+    table.add_column("Version")
+    table.add_column("State")
+
+    count = 0
+    async with client.apps.list(
+        cluster_name=cluster, org_name=org, project_name=project
+    ) as it:
+        async for app in it:
+            count += 1
+            table.add_row(
```

Comment:
Please use a formatter instead of direct making the table, as it is done here: https://github.com/neuro-inc/apolo-cli/blob/master/apolo-cli/src/apolo_cli/secrets.py#L80-L90

Later formatter could be modified to hide/show additional fields, etc. For example, jobs formatter has rich and complex logic: https://github.com/neuro-inc/apolo-cli/blob/master/apolo-cli/src/apolo_cli/formatters/jobs.py#L75

Formatter tests should be done by `rich_cmp(console)` usage, as here: https://github.com/neuro-inc/apolo-cli/blob/master/apolo-cli/tests/unit/formatters/test_jobs_formatters.py#L256

To regen ascii files use `pytest ... --rich-gen` option, then check diffs for generated files